src/library/scala/collection/concurrent/Map.scala

```diff
@@ -32,8 +32,6 @@ import scala.annotation.tailrec
   *  provides all of the methods a `Map` does, with the difference that all the
   *  changes are atomic. It also describes methods specific to concurrent maps.
   *
-  *  '''Note''': The concurrent maps do not accept `'''null'''` for keys or values.
-  *
```

Comment:
Why is the note about non supporting null deleted? This looks sneaky, does not seem to be related to the rename.

Reply:
https://github.com/scala/scala/commit/606fb73b88edfc4b3ce9a68289306acc4fa8cd2b